src/plugin.ts

```diff
@@ -4,9 +4,46 @@ import { createClient } from "@supabase/supabase-js";
 import { LogReturn, Logs } from "@ubiquity-os/ubiquity-os-logger";
 import { createAdapters } from "./adapters";
 import { userPullRequest, userSelfAssign, userStartStop, userUnassigned } from "./handlers/user-start-stop";
-import { Context, Env, PluginInputs } from "./types";
+import { Context, Env, GitHubIssueSearch, PluginInputs } from "./types";
 import { addCommentToIssue } from "./utils/issue";
 
+async function listOrganizations(context: Context): Promise<string[]> {
```

Comment:
Can you place this in with the handlers or helpers to keep the entry tidy my friend?

Reply:
sure. will do so